script/install_cuda.sh

```diff
@@ -138,16 +138,16 @@ then
         ALPAKA_CUDA_PKG_FILE_PATH=http://developer.download.nvidia.com/compute/cuda/11.0.3/local_installers/${ALPAKA_CUDA_PKG_FILE_NAME}
     elif [ "${ALPAKA_CI_CUDA_VERSION}" == "11.1" ]
     then
-        ALPAKA_CUDA_PKG_FILE_NAME=cuda_11.1.0_456.43_win10.exe
-        ALPAKA_CUDA_PKG_FILE_PATH=http://developer.download.nvidia.com/compute/cuda/11.1.0/local_installers/${ALPAKA_CUDA_PKG_FILE_NAME}
+        ALPAKA_CUDA_PKG_FILE_NAME=cuda_11.1.1_456.81_win10.exe
```

Comment:
I thought we're not testing CUDA 11.1?

Reply:
I have not removed it to allow maybe to bring it back if it is not making issues anymore. We have the same in the develop branch.

Reply:
Most changes are cherry picks, I try always to keep all unchanged, except there are merge conflicts.